engine/src/main/java/org/terasology/engine/core/PathManager.java

```diff
@@ -1,7 +1,7 @@
 // Copyright 2021 The Terasology Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-package org.terasology.engine.core.paths;
+package org.terasology.engine.core;
```

Comment:
oh yeah. All the import changes are because I decided that PathManager doesn't deserve its own sub-package. It was the only thing in `engine.core.paths`.